src/DataTable.js

```diff
@@ -76,6 +78,17 @@ export default function DataTable(props) {
             }}
         >
             <h4 className="chw-table-title">{title}</h4>
+            <span style={{ verticalAlign: "middle" }}>
```

Comment:
```suggestion
            <span style={{ verticalAlign: "middle", display: "inline-block" }}>
```

Reply:
It looks, without this change, a certain browser (i.e., Safari in my local) seems to position the button improperly?

Looks like we have to assign values to `display` if we want to specify the size (width/height) for the inner elements?